nextflow.config

```diff
@@ -16,9 +16,7 @@ params {
         tabix = "tabix/0.2.6"
         bedtools = "BEDTools/2.23.0"
     }
-    programs {
-        svvcf2bed = "/home/rajohvik/Work/wgs-structvar/SVvcf2bed.pl"
-    }
+    svvcf2bed = "/home/rajohvik/Work/wgs-structvar/SVvcf2bed.pl"
```

Comment:
Shouldn't this be updated now as we have the script in the repo (as per fbcda3)?